app/elements/candidates/invite-candidates.html

```diff
@@ -78,14 +74,13 @@
 				}
 			},
 			_addCandidate: function(){
-				if (this.challenge){
-					this.push('candidates', {'Challenge': this.challenge.Key});
+				if (this.challenges.length == 1){
+					this.push('candidates', {'Challenge': this.challenges[0].Key});
```

Comment:
Getting a warning that you did not set a challenge is better than having a default challenge set and it not being the right one.